src/tools/tidy/src/style.rs

```diff
@@ -62,10 +63,40 @@ const ANNOTATIONS_TO_IGNORE: &[&str] = &[
     "// normalize-stderr-test",
 ];
 
-// Intentionally written in decimal rather than hex
-const PROBLEMATIC_CONSTS: &[u32] = &[
-    184594741, 2880289470, 2881141438, 2965027518, 2976579765, 3203381950, 3405691582, 3405697037,
-    3735927486, 3735932941, 4027431614, 4276992702,
+fn generate_problems<'a>(consts: &'a[u32], letter_digit: &'a HashMap<char, char>) -> impl Iterator<Item = u32> + 'a {
```

Comment:
The code is required to be formatted by rustfmt. Please run `./x.py fmt`.

Reply:
I don't have access to a computer right now, wrote that from my phone... :(

Reply:
Will do that when i can